packages/components/src/Button.tsx

```diff
@@ -0,0 +1,95 @@
+import * as React from "react";
+import { Link } from "react-router-dom";
+import styled, { StyledComponentClass } from "styled-components";
+
+import { colors, fonts } from "./styleConstants";
+
+export interface ButtonProps {
+  className?: string;
+  icon?: any;
+  to?: string;
+  active?: boolean;
+  disabled?: boolean;
+  inputRef?: any;
+  name?: string;
+  onClick?(ev: any): any;
```

Comment:
if the result is not important you can use `onClick?(ev: any): void` and it will work